### PR TITLE
GEOMESA-1403 Fix GeoMesa Command Line tools docs typos and formatting…

### PR DESCRIPTION
**Pull request by aheyne**
… issues.

Fixed some inconsistent formatting and typos that prevented the examples from working.

Signed-off-by: Austin Heyne <aheyne@ccri.com>

### CODE REVIEW

File: docs/user/commandline_tools.rst [hunk 1]
```diff
@@ -216,11 +216,16 @@ includes some examples. SFT and Converter specifications should use the path pre
 
 For example, here's a simple CSV file to ingest named ``example.csv``::
 
-    ID,Name,Age,LastSeen,Friends,Lat,Lon
+    FID,Name,Age,LastSeen,Friends,Lat,Lon
```
Comment (mdzimmerman) [inline docs/user/commandline_tools.rst]: Good catch!

File: docs/user/commandline_tools.rst [hunk 2]
```diff
@@ -216,11 +216,16 @@ includes some examples. SFT and Converter specifications should use the path pre
...
     23623,Harry,20,2015-05-06,"Will, Mark, Suzan",-100.236523,23
     26236,Hermione,25,2015-06-07,"Edward, Bill, Harry",40.232,-53.2356
     3233,Severus,30,2015-10-23,"Tom, Riddle, Voldemort",3,-62.23
 
+.. note::
+
+    ID is a reserved word, for a full list of reserved words see
+    `Geomesa Data Store Reserved Words <http://www.geomesa.org/documentation/user/datastores.html#reserved-words>`__.
```
Comment (mdzimmerman) [inline docs/user/commandline_tools.rst]: This hardcodes the URL for the built docs, which will break a lot of things (versioned forms of old docs, non-HTML doc builds). Better to use ``:ref:`` (see http://www.sphinx-doc.org/en/stable/markup/inline.html#role-ref). E.g. put

    :ref:`reserved_words`

here, and

    .. _reserved_words:

above the Reserved Words section of ``datastores.rst``.

File: docs/user/commandline_tools.rst [hunk 3]
```diff
@@ -265,11 +270,11 @@ The SFT and Converter can be referenced by name and the following commands can i
 
     $ geomesa ingest -u username -p password \
       -c geomesa_catalog -i instance \
-      -s renegates -C renegades-csv example1.csv
+      -s renegades -C renegades-csv example.csv
```
Comment (mdzimmerman) [inline docs/user/commandline_tools.rst]: Wow, we just cannot spell that word properly!

**Comment by mdzimmerman** [comment]
Apart from the crossref, looks good to me.

**Comment by aheyne** [comment]
I've updated the cross ref.

**Comment by elahrvivaz** [comment]
+1

**Comment by cne1x** [comment]
Looks good to me.

**Pull request merged**